config/routes.rb

```diff
@@ -1,6 +1,13 @@
 Rails.application.routes.draw do
   # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
+  get "post", to: "post#index"
+  get "post", to: "post#show"
 
+  get "user", to: "user#index"
+  get "user", to: "user#show"
+
+  get "comment", to: "comment#index"
+  get "comment", to: "comment#show"
```

Comment:
@Irirwanirira kindly ensure your routes matches the routes given, the posts should belong to user and for you to go to show page you nedd first to view the index page take alook at the comparison below:

Routes Given          |  Your Routes    
:-------------------------:|:-------------------------:
![image](https://user-images.githubusercontent.com/61408860/215460902-04885fbe-881c-44ff-8cb6-8409e8dfd728.png)  |  ![image](https://user-images.githubusercontent.com/61408860/215461725-5f32bf94-56cd-4032-a1e7-446e029c80b3.png)

`It would be better if you used resources for your routes , this makes writing a short code and also easier to scale in future.` eg for view index page for posts and comments

``` 
 resources :posts, only: [:index] do
    resources :comments, only: [:index]
  end
```

Reply:
- [ ] Kindly change the controllers and views directory name they should be in prural eg posts users